features/step_definitions/run_steps.rb

```diff
@@ -58,8 +58,9 @@
 end
 
 
-Then(/^I see "(.*)"$/) do |string|
-  expect(@last_run_result.out).to include string
+Then(/^I(?: do|( don't))? see "(.*)"$/) do |negate, string|
+  method = negate ? :not_to : :to
```

Comment:
I'd like the "do" to be optional though.

"Then I see ..." for normal sentences.
"Then I do see ..." for scenario sentences.